algorand/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "@certusone/wormhole-sdk": "file:../sdk/js",
-    "algosdk": "^1.14.0",
+    "algosdk": "^1.21.0",
```

Comment:
Does this have any effect on sdk/js?  Like if tests pass here because 1.21.0 is being used, can same things fail in sdk/js because it is using older version?

Reply:
No, this should have no effect... but we might want to upgrade the algosdk used in our sdk...